velox/functions/lib/tests/DateTimeFormatterTest.cpp

```diff
@@ -2441,4 +2441,12 @@ TEST_F(SimpleDateTimeFormatterTest, formatWeekOfMonth) {
   }
 }
 
+TEST_F(SimpleDateTimeFormatterTest, parseUsingPartialInput) {
+  for (bool lenient : {true, false}) {
```

Comment:
nit: for easier debugging, we could write each test case separately.

Reply:
Updated.